### PR TITLE
Jormun: fix types sent to bragi

### PR DESCRIPTION
**Pull request by datanel**
|     jormungandr type   |   bragi type |
| :-------------: | :---------: |
|        stop_area        |      public_transport:stop_area |
|        address        |      street + house |
|        administrative_region        |     city |
|        poi        |     poi |

### CODE REVIEW

**Review by TeXitoi — APPROVED** [review]

File: source/jormungandr/jormungandr/autocomplete/geocodejson.py [hunk 1]
```diff
@@ -64,12 +70,15 @@ def make_params(self, request, instance):
...
-                    type = 'city'
-
-                types.append(type)
+                    types.append(self.TYPE_CITY)
+                elif type == 'address':
+                    types.extend([self.TYPE_STREET, self.TYPE_HOUSE])
+                elif type == 'stop_area':
+                    types.append(self.TYPE_STOP_AREA)
+                else:
+                    types.append(type)
```
Comment (azime) [inline source/jormungandr/jormungandr/autocomplete/geocodejson.py]:
```python
map_type = {
"administrative_region": ["city"],
"address": ["house", "street"],
"stop_area": ["public_transport:stop_area"],
"poi": ["poi"]
}
```
and 
```python
types.extend(map_type[type])
```

Comment (datanel) [inline source/jormungandr/jormungandr/autocomplete/geocodejson.py]: seems not working 
It concatenates each letter
`[u'p', u'u', u'b', u'l', u'i', u'c', u'_', u't', u'r', u'a', u'n', u's', ....`

Comment (datanel) [inline source/jormungandr/jormungandr/autocomplete/geocodejson.py]: ok works with `types.extend([map_type[type]])`

**Comment by datanel** [comment]
do you think that the constants are necessary?

**Comment by antoine-de** [comment]
I wouldn't have added the constants but it's fine to keep them

**Pull request merged**